numerics/elliptic_integrals_test.cc

```diff
@@ -1,316 +1,31 @@
-
+﻿
+#include <filesystem>
 #include <vector>
```

Comment:
This file should have the cpp extension.

Reply:
Amusing.